translate/src/modules/user/components/UserAvatar.tsx

```diff
@@ -5,10 +5,12 @@ type Props = {
   username: string;
   title?: string;
   imageUrl: string;
+  userStatus: string | string[];
```

Comment:
`userStatus` should actually always be `string[]`:

```suggestion
  userStatus: string[];
```

To make the `make types` pass, you'll also need to fix the original sin - the type set in both files in the `api` folder.